qiita_pet/static/js/sharing.js

```diff
@@ -0,0 +1,56 @@
+var current_study = null;
```

Comment:
From offline conversation, @ElDeveloper and I agree it's not feasible to remove this global variable at this time.